### PR TITLE
Add "gogoimport" option to protoc-gen-govalidators command-line.

### PR DESCRIPTION
**Pull request by devnev**

### CODE REVIEW

**Comment by mwitkow** [comment]
Please add a README section in `Installing and using`

File: protoc-gen-govalidators/main.go [hunk 1]
```diff
@@ -29,12 +30,26 @@ func main() {
 		gen.Fail("no files to generate")
 	}
 
+	useGogoImport := true
```
Comment (mwitkow) [inline protoc-gen-govalidators/main.go]: default to false? :)

**Comment by mwitkow** [comment]
ROFL

**Comment by devnev** [comment]
PTAL.

File: plugin/plugin.go [hunk 2]
```diff
@@ -58,24 +58,26 @@ import (
 	"github.com/gogo/protobuf/proto"
 	descriptor "github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+	"github.com/gogo/protobuf/vanity"
 	"github.com/mwitkow/go-proto-validators"
 )
 
 func init() {
-	generator.RegisterPlugin(NewPlugin())
+	generator.RegisterPlugin(NewPlugin(true))
```
Comment (mwitkow) [inline plugin/plugin.go]: Is this even need this at all? We do `generator.New` in the main anyway.

Comment (devnev) [inline plugin/plugin.go]: We don't really. It's registering as a plugin with the main gogo generator, but we never even invoke that.

**Comment by mwitkow** [comment]
Thanks!

**Pull request merged**